tests/cases/mapInteractor.js

```diff
@@ -1691,4 +1691,131 @@ describe('mapInteractor', function () {
     interactor.simulateEvent('keyboard', {keys: '2'});
     expect(triggered).toBe(0);
   });
+
+  it('Test touch interactions', function () {
+    var map = mockedMap('#mapNode1'),
+        interactor = geo.mapInteractor({map: map});
+
+    expect(interactor.hasTouchSupport()).toBe(true);
+
+    // check the pan event was called
+    interactor.simulateEvent(
+      'panstart', {touch: true, center: {x: 20, y: 20}});
+    interactor.simulateEvent(
+      'panmove', {touch: true, center: {x: 30, y: 20}});
+    interactor.simulateEvent(
+      'panend', {touch: true, center: {x: 40, y: 20}});
+    expect(map.info.pan).toBe(2);
+    expect(map.info.panArgs.x).toBe(10);
+    expect(map.info.panArgs.y).toBe(0);
+
+    // A two-pointer event will end the action
+    interactor.simulateEvent(
+      'panstart', {touch: true, center: {x: 20, y: 20}});
+    interactor.simulateEvent(
+      'hammer.input', {touch: true, center: {x: 30, y: 20}, pointers: [1]});
```

Comment:
Simulates the number of fingers down.  A two-touch gesture where you then release one figure will result in a following one-touch gesture unless we stop it.  The pointers is a list of those touches, but we don't care what is in them, just how many there are, so for the test it can be any list of the appropriate length.

If you don't do this, a two-touch zoom where you let up on one figure first will result in a pan which isn't desired.